### PR TITLE
Updated phragmen bounds param

### PR DESCRIPTION
**Pull request by ToufeeqP**

### CODE REVIEW

File: runtime/src/constants.rs [hunk 1]
```diff
@@ -286,9 +281,9 @@ pub mod staking {
 		// Note: the EPM in this runtime runs the election on-chain. The election bounds must be
 		// carefully set so that an election round fits in one block.
 		pub ElectionBoundsMultiPhase: ElectionBounds = ElectionBoundsBuilder::default()
-			.voters_count(10_000.into()).targets_count(1_500.into()).build();
+			.voters_count(MaxElectingVoters::get().into()).build();
 		pub ElectionBoundsOnChain: ElectionBounds = ElectionBoundsBuilder::default()
-			.voters_count(5_000.into()).targets_count(1_250.into()).build();
+			.voters_count(5_000.into()).build();
```
Comment (fmiguelgarcia) [inline runtime/src/constants.rs:286]: Could we also define `5_000` as a constant too?

File: runtime/src/impls.rs [hunk 2]
```diff
@@ -455,7 +455,7 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type Fallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
 	type ForceOrigin = EnsureRoot<AccountId>;
 	type GovernanceFallback = onchain::OnChainExecution<OnChainSeqPhragmen>;
-	type MaxWinners = constants::MaxActiveValidators;
+	type MaxWinners = constants::staking::MaxActiveValidators;
```
Comment (fmiguelgarcia) [inline runtime/src/impls.rs:458]: Could we remove `constants::MaxActiveValidators` if that is not used anymore?

Comment (ToufeeqP) [reply runtime/src/impls.rs:458]: Its already removed

File: runtime/src/constants.rs (hunk 1 above)
Comment (ToufeeqP) [reply runtime/src/constants.rs:286]: yeah, will do that

**Review by markopoloparadox — APPROVED** [review]

**Review by fmiguelgarcia — APPROVED** [review]

**Pull request merged**